app/presenters/instance_presenter.rb

```diff
@@ -25,4 +24,10 @@ def status_count
   def domain_count
     Rails.cache.fetch('distinct_domain_count') { Account.distinct.count(:domain) }
   end
+
+  def open_registrations
```

Comment:
Add some spec coverage for this method to show that various combination of settings and user counts are handled correctly?